CMakeLists.txt

```diff
@@ -1,5 +1,43 @@
-project(libexecstream)
-cmake_minimum_required(VERSION 2.8)
+project( libexecstream_prj )
```

Comment:
Please leave it as libexecstream, or libexecstream_project if you want to make a distinction from the library target itself